pytest_inmanta/plugin.py

```diff
@@ -647,6 +647,9 @@ def _load(self) -> None:
         test_project = module.Project(self._test_project_dir)
         module.Project.set(test_project)
         test_project.load()
+        # refresh plugins
+        if self._should_load_plugins is not None:
+            self._plugins = self._load_plugins()
```

Comment:
This might introduce significant overhead if a lot of tests use the `project` fixture. If we merge this in we'll need to keep an eye on this.